report/src/main/webapp/app/components/keyboardShortcuts.directive.js

```diff
@@ -141,36 +142,41 @@ define(['angularAMD', 'userSettingsService'], function (angularAMD) {
             if (!_.isEmpty($(suiteContainer).nextAll(
                     '.aside-report:not(.is-hidden)').first().find(
                     '.test-name'))) {
+              currentTab = $('.nav-tabs > .nav-item').filter('.active').text().replace(/\s/g, '');
               testContainer.removeClass('is-active');
               currentItem.removeClass('is-active');
-              toggleNextTest(suiteContainer);
               $nextElement = suiteContainer.nextAll(
                   '.aside-report:not(.is-hidden)').first();
               $nextElement.addClass('is-expanded');
               $nextElement.children().first().addClass('is-active');
               scrollTo($nextElement.find('.is-active'));
+              toggleNextTestItem(suiteContainer);
+              userSettingsService.setLastTab(currentTab);
             }
-
           } else {
+            currentTab = $('.nav-tabs > .nav-item').filter('.active').text().replace(/\s/g, '');
```

Comment:
Please add a `$` prefix to the variable name to indicate it holds a jQuery object

Reply:
Even though it's just plain text at the end ('screenlayout' for example)?

Reply:
Ok, didn't notice - no need to worry about it then.

Reply:
The name is very misleading though - rename it to `currentTabLabel` or something else more accurate